calico/common.py

```diff
@@ -35,6 +35,8 @@
 from types import StringTypes
 from netaddr.strategy import eui48
 
+MESSAGE_LOG_NAME = "message_tracking_log"
```

Comment:
This should be under "calico.".  You could even put it under the actor namespace at "calico.felix.actor.messages", for example.